packages/ripple-address-codec/package.json

```diff
@@ -21,12 +21,12 @@
   "prepublishOnly": "tslint -b ./ && jest",
   "scripts": {
     "build": "tsc -b",
-    "test": "jest",
+    "test": "jest --verbose false --silent=false --runInBand ./src/*.test.js",
```

Comment:
Why does this need to change?

Reply:
Was just adding consistency among all packages, happy to change if you want

Reply:
I'd prefer to avoid `--runInBand` where possible, so I think it's better to leave it as just `jest`.

Reply:
Fixed